src/helpers.ts

```diff
@@ -162,13 +162,27 @@ export class Helpers {
     return array;
   }
   public static getUnbindValue(value: any): any {
```

Comment:
Hey @andrewtelnov thanks for the fix. But I wonder what this function tries to achieve?

Reply:
@salargl  If you run unit tests with your code then 68 unit tests will be failed.
Here is the reason:
```js
const val = matrixQustion.value;
val.push(newValue); //this line will modify matrixQustion.value becaue the instance is the same.
```
I had to modify the code to the following to fix unit tests in SurveyJS Creator V1&V2:
```js
  public static getUnbindValue(value: any): any {
    if(Array.isArray(value)) {
      const res = [];
      for(let i = 0; i < value.length; i ++) {
        res.push(Helpers.getUnbindValue(value[i]));
      }
      return res;
    }
    if (!!value && value instanceof Object && !(value instanceof Date)) {
      return JSON.parse(JSON.stringify(value));
    }
    return value;
  }

```

Thank you,
Andrew